JS_lesson7/Task3.js

```diff
@@ -0,0 +1,24 @@
+// Task 3
+
+// 5*. Напишите функцию findMissingNumber(arr), которая принимает массив чисел от 1 до N (исключая одно число) 
+//   и возвращает пропущенное число. Массив не отсортирован и может содержать дубликаты. 
+//   Решите эту задачу, используя эффективные методы массива.
+const arr = [1, 2, 9, 13, 8, 9, 3, 4, 5, 6, 10, 11, 12, 17, 16, 15, 11, 12 ]
+function findMissingNumber(arr) {
```

Comment:
Try with this case `const arr = [ 1, 2, 3, 4, 5, 6, 7]` - there is no missing number
You have return `null` for example